dali/sasha/saxref.cpp

```diff
@@ -2263,17 +2263,22 @@ class CSashaExpiryServer: public ISashaServer, public Thread
     Semaphore stopsem;
     Mutex runmutex;
     Owned<IUserDescriptor> udesc;
+    Linked<IPropertyTree> props;
 
 public:
     IMPLEMENT_IINTERFACE;
 
-    CSashaExpiryServer()
-        : Thread("CSashaExpiryServer")
+    CSashaExpiryServer(IPropertyTree *_config)
+        : props(_config), Thread("CSashaExpiryServer")
     {
         stopped = false;
 
         StringBuffer userName;
+#ifdef _CONTAINERIZED
+        serverConfig->getProp("@user", userName);
```

Comment:
From the helm chart I think this should be props->

Reply:
in fact, they're both the same in containerized mode (see code in createSashaFileExpiryServer)
Only in bare-metal is there an outer config level with the separate services (like DfuExpiry) underneath them.
However, it should refer to the passed in props for consistency with others, not directly to global serverConfig.

Will fix and squash